dev/devicelab/bin/tasks/plugin_lint_mac.dart

```diff
@@ -533,3 +533,24 @@ void _validateMacOSPodfile(String appPath) {
     'macos',
   ));
 }
+
+Future<void> _tryMacOSLint(String executable, List<String> arguments) async {
```

Comment:
We should move the executable and the first few arguments into this method, and make `arguments` into `podspecPath` + `extraArguments`. Right now, despite the name it can actually run any command, but then it's looking for `pod lint`-specific output.

Reply:
Good idea. Done!